e2e/mock-reset/__tests__/index.test.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
this doesn't have to be an integration test, I think adding it as a unit test to `jest-mock` is enough: https://github.com/facebook/jest/blob/aef5497c6685e8dd514e3f9ce9c601d03ca3cae2/packages/jest-mock/src/__tests__/index.test.ts#L1235

Reply:
I did e2e because @cpojer asked to. So it's decided do it as unit test and removing this e2e?

Reply:
yeah, I think a unit test makes more sense in this case

Reply:
@SimenB Done!